### PR TITLE
Enable allure back

### PR DESCRIPTION
**Pull request by arsentiy67**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -88,18 +75,18 @@
             <version>5.0.3</version>
         </dependency>
 
-        <dependency>
-            <groupId>io.rest-assured</groupId>
```
Comment (AkioFaris) [inline pom.xml:92]: why did you remove rest-assured?

Comment (Inov92) [reply pom.xml:92]: It was a duplicate

Comment (AkioFaris) [reply pom.xml:92]: found that you removed a duplicated, ok

**Review by AkioFaris — APPROVED** [review]
